src/store/index.js

```diff
@@ -22,4 +22,18 @@ export function getMessageSubject(message: Message): ?string {
   return subjectHeader && subjectHeader.value;
 }
 
+export function getMessageSender(message: Message): ?string {
+  const { headers } = message.payload;
+  const senderHeader = headers.find(header => header.name === 'From');
+  return senderHeader && senderHeader.value;
+}
```

Comment:
Not sure if I should admit this but I prefer your approach here versus [what I did](https://github.com/dkharlan/email-client-practice-app/blob/4e54773090ffd060c6012f6ee2b1ceb7cd07b2fd/src/ui/transform.js#L6) :joy: